src/services/comment.js

```diff
@@ -1,13 +1,25 @@
 const Comment = require('~/models/comment')
+const Cooperation = require('~/models/cooperation')
+const { createForbiddenError } = require('~/utils/errorsHelper')
 
 const commentService = {
   addComment: async (data) => {
     const { text, author, authorRole, cooperationId } = data
 
+    const { initiator, receiver } = await Cooperation.findById(cooperationId)
```

Comment:
What happens if a client provides non-existent `cooperationId`? Has it been checked before?

Reply:
When a client provides **a non-existent** ```cooperationId``` to create a new comment on this cooperation (using the _POST method_), they receive the following error:
<img width="1435" alt="Screenshot 2024-04-30 at 14 47 39" src="https://github.com/Space2Study-UA-1156/BackEnd-02/assets/28622400/82d182a2-a020-4915-a891-1a9ae90ebac3">
Also, when a client provides **a non-valid format** for ```cooperationId```, they receive the following error:
<img width="1435" alt="Screenshot 2024-04-30 at 14 46 22" src="https://github.com/Space2Study-UA-1156/BackEnd-02/assets/28622400/eb8522e1-5f77-4bea-957c-8dbac96ef9c5">
The same situation occurs when attempting to get a comment on **a non-existent** or **improperly formatted** ```cooperationId``` (using the GET method).

_These errors are handled by the following middleware:_
<img width="915" alt="Screenshot 2024-04-30 at 14 58 22" src="https://github.com/Space2Study-UA-1156/BackEnd-02/assets/28622400/ead3524e-960b-4062-8d7d-d298f46d6afc">
![Uploading Screenshot 2024-04-30 at 15.03.29.png…]()

Reply:
Thanks a lot!